RCovidSummary.R

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env Rscript
+# Author: JLinares/2022
+# Usage: 1. analysis summary for PatientSamples: Two files are generated: Summary and RunParameters
+#        2. Arguments: run name
+#        3. Example: Rscript / . . . ./summary_v2.R UT-A01290-220416
```

Comment:
Just a little thing; I had to look for a run that was still in the NGS. I used UT-A01290-220706.

Reply:
This line needs to be updated to 
```
Example: Rscript / . . . ./RCovidSummary.R UT-A01290-220416
```